### ISSUE TITLE
On/off telemetry direct by taranis 

### ISSUE THREAD

**Issue by jack113113** [openLRSng/openLRSng#212]
hello developers

Can we On/off telemetry easy by Taranis ?  then bind again ? in the fact , it is little inconvinience when us Configutor to this work

**Comment by mikeller** [comment]
@jack113113: Can you please elaborate a bit what user interaction and resulting behaviour you would like to see implemented?

**Comment by jack113113** [comment]
mikeller 

it same somthing here  , see photo 

![images](https://user-images.githubusercontent.com/35300588/38650204-cf8ca872-3e24-11e8-93a7-6d32e34cff6c.jpg)

**Comment by mikeller** [comment]
This is already implemented if you use the multiprotocol protocol on the TX module. You can set output power (high / low), with the value of low set by the 'option' parameter, and just disabling telemetry in the telemetry menu of your TX will disable telemetry.

**Comment by jack113113** [comment]
@mikeller 

Off telemetry on Taranis not display , but it does not Off telemetry from modul ,right ? 
i want to Off telemetry from modul .

if use configuration , then put it on Taranis with setup diffirent , so which it will work with setup  ?

**Comment by jack113113** [comment]
i tried  to change option 0 to  7  in power RF , but maybe it not change . 
Old version Firmware taranis have not OFF  telemetry . right ? i do not see in menu

**Comment by jack113113** [comment]
@mikeller  
i only can On/off Telemetry in ''' BIND ''  in internal Modul  Frysky Taranis . 
External modul Multi OLRS  , status '' Bind''  NOT choosen " ON/OFF TELEMETRY same frsky 2.4ghz 
I use OPENTX 2.2.1 x9d . I do not see where to OFF telemetry with external modul 

Can you support this problem ?

**Comment by jack113113** [comment]
@mikeller 
i real want to OFF telemetry direct On taranis by Bind  that do not need use Configulator . I want to better range .

**Comment by mikeller** [comment]
@jack113113: I still fail to see how and why you think that disabling telemetry on the TX will give you better range?

**Comment by jack113113** [comment]
😢😭😭😭😭

**Comment by geofrancis** [comment]
@mikeller disabling telemetry doubles the rc update speed, this allows you to use a lower radio rate giving you longer range. back when this was the "undead mod" for maximum range it ran at 8ch @ 4800 with no telemetry giving up to 8km on 100mw hardware.

**Comment by mikeller** [comment]
@geofrancis: Disabling telemetry **on the RX** does. Disabling telemetry on the TX will not affect how the link works at all, since the TX only reads the telemetry data.

**Comment by geofrancis** [comment]
yes thats the point he is getting at, the ability to disable the telemetry from the receiver from the handset. via the binding menu.

**Comment by mikeller** [comment]
I still don't understand. What you call the 'binding menu' can be used to change settings on the *TX*. This is different to settings on the *RX*. If user want to perform configuration of the RX, (or change all of the settings on the TX) from the TX, then then they should write a lua script that implements the configuration function of the openLRSng configurator.

**Comment by geofrancis** [comment]
but telemetry is set by the Tx module, then the receiver gets those settings during binding.

**Comment by mikeller** [comment]
@geofrancis: Please read https://github.com/openLRSng/openLRSng/issues/212#issuecomment-397774665. Changing the TX settings through the serial port is already possible, it just has to be implemented in OpenTX.
